README.md

```diff
@@ -1,16 +1,16 @@
 <h1 align="center">
-  <img style="vertical-align:middle" width="44" src="https://raw.githubusercontent.com/wxt-dev/wxt/HEAD/docs/public/hero-logo.svg" alt="WXT Logo">
+  <img align="top" width="44" src="https://raw.githubusercontent.com/wxt-dev/wxt/HEAD/docs/public/hero-logo.svg" alt="WXT Logo">
```

Comment:
Haha thanks I've wanted to fix that for a long time! I thought I tried `vertical-align`, but maybe I tried using it as an attribute? Cool that the style works here.